src/charm.py

```diff
@@ -35,6 +39,7 @@ class ContentCacheCharm(CharmBase):
     """Charm the service."""
 
     on = IngressCharmEvents()
+    loki_log_path = "/var/log/nginx/access.log"
```

Comment:
AFAIK loki supports multiple paths. We should be exporting not only the access log